lib/service/service.go

```diff
@@ -1744,7 +1785,7 @@ func (process *TeleportProcess) newLocalCacheForWindowsDesktop(clt auth.ClientI,
 // newLocalCache returns new instance of access point
 func (process *TeleportProcess) newLocalCache(clt auth.ClientI, setupConfig cache.SetupConfigFn, cacheName []string) (*cache.Cache, error) {
 	return process.newAccessCache(accessCacheConfig{
-		inMemory:  process.Config.CachePolicy.Type == memory.GetName(),
+		inMemory:  process.Config.CachePolicy.Type != lite.GetName(),
```

Comment:
Any reason to change this? If there is ever another option added besides memory/lite this will break

Reply:
Plenty of tests will fire up a `TeleportProcess` with a zero value `CachePolicy`, so they were using the default by accident before. I'll call `CheckAndSetDefaults` on the `CachePolicy` error out if that fails or if the type is not one of the two that `newAccessCache` supports (`in-memory` and `sqlite`).